lib/mix/tasks/gen/ash.gen.expression.ex

```diff
@@ -0,0 +1,99 @@
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.Ash.Gen.Expression do
```

Comment:
We should call this `ash.gen.custom_expression` since that is what they are called.

Reply:
So should everything have that prefix? `ash.gen.custom_change`

Reply:
Oh, it's only for expression. Alright.